homework.md

```diff
@@ -44,70 +44,72 @@ Using CSS, how would I select every `<h1>` tag on the page?
 
 What is JavaScript?
 
-A programming language used on the internet
+A programming language of the web
```

Comment:
Hey Waleed, Although its fine, would you mind writing and explaining a bit more about this question. The rests seems fine to me. Nice work!

Reply:
hey @Waleed, Great work keep up the good work.